modules/wallet/persist_legacy.go

```diff
@@ -1,177 +0,0 @@
-	for _, skey := range savedKeys {
-		// Skip this key if it's already known to the wallet.
-		_, exists := w.keys[skey.UnlockConditions.UnlockHash()]
-		if exists {
-			continue
-		}
-
-		// Create an entry in w.keys for each savedKey.
-		w.keys[skey.UnlockConditions.UnlockHash()] = &key{
-			spendable:        height >= skey.UnlockConditions.Timelock,
-			unlockConditions: skey.UnlockConditions,
-			secretKey:        skey.SecretKey,
-			outputs:          make(map[types.SiacoinOutputID]*knownOutput),
-		}
-
-		// If Timelock != 0, also add to set of timelockedKeys.
-		if tl := skey.UnlockConditions.Timelock; tl != 0 {
-			w.timelockedKeys[tl] = append(w.timelockedKeys[tl], skey.UnlockConditions.UnlockHash())
-		}
-
-		if skey.Visible {
-			w.visibleAddresses[skey.UnlockConditions.UnlockHash()] = struct{}{}
-		}
-	}
-
-	// If there are no visible addresses, create one.
-	if len(w.visibleAddresses) == 0 {
-		_, _, err := w.coinAddress(true)
-		if err != nil {
-			return err
-		}
-	}
-
-	return nil
-}
-
-// loadWallet pulls a wallet from disk into memory, merging it with whatever
-// wallet is already in memory. The result is a combined wallet that has all of
-// the addresses.
-func (w *Wallet) loadWallet(filepath string) error {
-	var savedKeys []savedKey
-	err := encoding.ReadFile(filepath, &savedKeys)
-	if err != nil {
-		return err
-	}
-	err = w.loadKeys(savedKeys)
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
-// loadSiafundTracking loads siafund addresses for tracking.
-func (w *Wallet) loadSiafundTracking(filepath string) error {
-	// Load the siafunds file, which is intentionally called 'outputs.dat'.
-	var siafundAddresses []types.UnlockHash
-	err := encoding.ReadFile(filepath, &siafundAddresses)
-	if err != nil {
-		return err
-	}
-
-	// Load the addresses into the wallet.
-	for _, sa := range siafundAddresses {
-		w.siafundAddresses[sa] = struct{}{}
-	}
-	return nil
-}
-
-// load reads the contents of a wallet from a file.
-func (w *Wallet) load() error {
-	err := w.loadWallet(filepath.Join(w.persistDir, "wallet.dat"))
-	if err != nil {
-		// try loading the backup
-		// TODO: display/log a warning
-		err = w.loadWallet(filepath.Join(w.persistDir, "wallet.backup"))
-		if err != nil {
-			return err
-		}
-	}
-
-	// Load the siafunds file, which is intentionally called 'outputs.dat'.
-	err = w.loadSiafundTracking(filepath.Join(w.persistDir, "outputs.dat"))
-	if err != nil {
-		// try loading the backup
-		// TODO: display/log a warning?
-		err = w.loadSiafundTracking(filepath.Join(w.persistDir, "outputs.backup"))
-		if err != nil {
-			return err
-		}
-	}
-
-	return w.save()
-}
-
-// MergeWallet merges another wallet with the already-loaded wallet, creating a
-// new wallet that contains all of the addresses from each. This is useful for
-// loading backups.
-func (w *Wallet) MergeWallet(filepath string) error {
-	err := w.loadWallet(filepath)
-	if err != nil {
-		return err
-	}
-	return w.save()
-}
-*/
```

Comment:
Noted, I'll reference it when I write the compatibility code. But there isn't much sense in keeping it around if it's all commented out, especially since it will be 100% obsolete in a few weeks.

Speaking of which, when I write the compat code, I should use the new approach of defining functions in the `compatibility` package, right?